presentation/src/main/java/com/example/pinterbest/presentation/ActualPinFragment.kt

```diff
@@ -69,20 +99,19 @@ class ActualPinFragment : Fragment() {
             }
         }
         viewModel.state.observe(
-            viewLifecycleOwner,
-            { loading ->
-                when (loading) {
-                    true -> {
-                        binding.actualPinScreen.visibility = View.GONE
-                        binding.emptyView.visibility = View.GONE
-                        binding.progressBar.visibility = View.VISIBLE
-                    }
-                    false -> {
-                        binding.progressBar.visibility = View.GONE
-                    }
+            viewLifecycleOwner
+        ) { loading ->
+            when (loading) {
+                true -> {
+                    binding.actualPinScreen.visibility = View.GONE
+                    binding.emptyView.visibility = View.GONE
```

Comment:
don't you need to make `actualPinScreen` and `emptyView` visible when `loading` is false?